roles/openshift_gce_image_prep/tasks/main.yaml

```diff
@@ -0,0 +1,10 @@
+# GCE instances are starting with xfs AND barrier=1, which is only for extfs.
+- name: Remove barrier=1 from XFS fstab entries
+  command: sed -i -e 's/xfs\(.*\)barrier=1/xfs\1/g; s/, / /g' /etc/fstab
```

Comment:
lineinfile is probably the ansible way.

Reply:
Can line in file correctly replace and add?  I wasn't able to find how to guarantee that.

Reply:
Updated, didn't see backrefs before